docs/geedocs/5.2.5/answer/176738.html

```diff
@@ -256,7 +256,7 @@ <h2 id="cache_configuration">Cache configuration</h2>
 The asset cache size is controlled by <code>AssetCacheSize</code> and the asset version cache size is controlled by <code>VersionCacheSize</code>.
 Note that the asset version cache is generally larger than the asset cache because it may contain multiple versions of the same asset.</p>
 
-Also included within <code>misc.xml</code> is an experimental option, which allows for the user to disable cache purging while projects build. This is controlled by <code>DisablePacketLevelVersionCachePurge</code> and its default value is <code>false</code>.
+Also included within <code>misc.xml</code> is an experimental option, which allows for the user to disable cache purging while projects build. This is controlled by <code>DisablePacketLevelVersionCachePurge</code> and its default value is <code>false</code>
```

Comment:
Need to remove this here since there's a separate PR.

Reply:
This is what was originally in the 5.2.5 doc file, this is not the new addition. It appears I removed a period at the end of the original by mistake.